doc/installation.rst

```diff
@@ -133,9 +133,7 @@ The requirements to run Spyder are:
 
 * `Python <http://www.python.org/>`_ 2.7 or >=3.3
 
-* `PyQt5 <https://www.riverbankcomputing.com/software/pyqt/download5>`_ >=5.2 or
-  `PyQt4 <https://www.riverbankcomputing.com/software/pyqt/download>`_ >=4.6.0
-  (PyQt5 is recommended).
+* `PyQt5 <https://www.riverbankcomputing.com/software/pyqt/download5>`_ >=5.5
```

Comment:
Doc changes; see top level comment.

Reply:
@ccordoba12 Just FYI, if you don't get to it before I do my pass for baseline correctness updates (which will happen as soon as we resolve spyder-ide/spyder-docs#14 ) I can just do it as one part of that, as it is a close fit for the purpose and scope of the issue its resolving.